content/en/account_management/authn_mapping/_index.md

```diff
@@ -222,7 +231,14 @@ Replace the `<YOUR_DATADOG_API_KEY>` and `<YOUR_DATADOG_APPLICATION_KEY>` placeh
       }
     }
   ],
-  "meta": {"page": {"total": 0, "page_number": 0, "page_size": 0}}
+  "meta": {
+    "page": {
+      "total_count": 1, 
+      "total_filtered_count": 1, 
+      "page_number": 0, 
+      "page_size": 0
```

Comment:
is this consistent with the users + roles API?

Reply:
Users and roles API do not have a page_number and page_size, the other two fields are correct. I can remove them if they're causing confusion. AFAIK they're not being used by the frontend.